packages/repack/src/webpack/loaders/assetsLoader.ts

```diff
@@ -112,6 +112,9 @@ export default async function reactNativeAssetsLoader(this: LoaderContext) {
         parseInt(b.replace(/[^\d.]/g, ''), 10)
     );
 
+    const scaleNumbers = scaleKeys.map((scale) =>
+      parseInt(scale.replace(/[^\d.]/g, ''), 10)
```

Comment:
Should not this have been a parseFloat? In the switch statement on row 140 you expect resolutions to be able to come in as floats. Not sure that is actually used in the wild. But if they do this could be a potentail bug   @zamotany

Reply:
Thanks :) you're right. I'll fix it 👍

Reply:
https://github.com/callstack/repack/pull/156